artview/components/limits.py

```diff
@@ -120,7 +120,7 @@ def _default_limits(field, plot_type):
 ###############################
```

Comment:
current aspect ratio is defined as auto, I would be careful on changing that resizing may become problematic.